### DISCUSSION TITLE
自分が投稿した記事のみを検索する機能

### DISCUSSION THREAD

**Discussion by ghost** [increments/qiita-discussions#359] [要望・アイデア・不具合]
投稿記事を100件超えたところから、自分の特定の記事を見たい時、検索機能があると便利だと思うことが多々あります。
そのため、マイページから自分が投稿した記事のみを検索する機能があれば助かります

**Comment by shiracamus** [comment]
ヘルプページに書いてある検索オプションの user:ユーザ名 を付けて検索してみてはいかがでしょうか？
・検索時に利用できるオプション
https://help.qiita.com/ja/articles/qiita-search-options

**Reply by ghost** [reply]
@shiracamus
オプションで絞ることができました。ありがとうございます！

**Comment by tanaka-asami415** [comment]
@hiraiyuji 
解決されたようでよかったです！
同様のご意見はいただいており、マイページを改善していきたいとは思っているので、
その際に今回頂いた内容も検討させていただければと思います！
お手数をおかけしてすみませんが、いまのところは検索画面等で対応いただければ幸いです🙇

@shiracamus 
代わりに回答いただきありがとうございます！🙇

**Comment by gilly135** [comment]
他のユーザーの方からのリクエストもあり、ユーザーページから、閲覧中のユーザーを検索条件に指定した検索結果ページに遷移できるようにしました！

詳細はリリースノートをご確認ください。
https://qiita.com/release-notes/2024-07-24

GitHubのアカウントを削除なさっているので、今回の対応を持ちまして本件Closeとさせていただきます。
またご要望がございましたら、お知らせいただけると嬉しいです 🙏
